packages/api/pages/api/languages/[code]/import.ts

```diff
@@ -0,0 +1,28 @@
+import { PostLanguageImportRequestBody } from '@translation/api-types';
+import { z } from 'zod';
+import createRoute from '../../../../shared/Route';
+import { authorize } from '../../../../shared/access-control/authorize';
+
+export default createRoute()
+  .post<PostLanguageImportRequestBody, void>({
+    authorize: authorize((req) => ({
+      action: 'administer',
+      subject: 'Language',
+      subjectId: req.query.code,
+    })),
+    schema: z.object({
+      import: z.string(),
+    }),
+    async handler(req, res) {
+      // TODO: implement import logic.
+      console.log('import language:', req.body.import);
+      // simulate the processing time
+      await new Promise<void>((resolve) => {
+        setTimeout(() => {
+          resolve();
+        }, 10000);
+      });
+      res.ok();
```

Comment:
If you add a `throw Error();` above the `res.ok`, you can test that the error flash shows up correctly